src/features/profile/ProfileCard.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
```

Comment:
@reacteer the profile card component is used for different pages/screens so it should be inside `/src/common/components`